docs/samples/browser-plugin-meetings/index.html

```diff
@@ -19,6 +19,7 @@ <h2 class="collapsible">Multistream Media</h2>
           <audio id="multistream-remote-audio-0" class="multistream-remote-audio" autoplay ></audio>
           <audio id="multistream-remote-audio-1" class="multistream-remote-audio" autoplay ></audio>
           <audio id="multistream-remote-audio-2" class="multistream-remote-audio" autoplay ></audio>
+          <audio id="multistream-remote-share-audio" autoplay ></audio>
```

Comment:
Question: Don't we need an equivalent for video too?

Reply:
no, because video elements are created dynamically depending on the selected layout. The screen share ones are created in `createScreenshareView()`